### PR TITLE
n17: Vollmond

### PR DESCRIPTION
**Pull request by puppetmaster-**
#408

### CODE REVIEW

File: content/posts/newsletter-017/index.md [hunk 1]
```diff
@@ -85,6 +85,35 @@ This month's updates:
 [wor-discord]: https://discord.gg/JGeVt5XwPP
 [wor-website]: https://anthropicstudios.com/way-of-rhea
 
+### [Vollmond][vollmond-itch]
+
+![Vollmond preview](vollmond-preview.png)
+
+[Vollmond][vollmond-itch] a small non violent adventure in the style of Zelda 2.
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-017/index.md:92]:
```suggestion
[Vollmond][vollmond-itch] is a small non-violent adventure in the style of Zelda 2.
```

**Review by 17cupsofcoffee — APPROVED** [review]
:+1: Looks good, just one minor grammar suggestion:

File: content/posts/newsletter-017/index.md (hunk 1 above)
Comment (puppetmaster-) [reply content/posts/newsletter-017/index.md:92]: I agree. Do you want me to change it or do you?

Comment (17cupsofcoffee) [reply content/posts/newsletter-017/index.md]: I've applied the change :+1:

**Review by ozkriff — APPROVED** [review]
:fire: (made a few small list style tweaks)

**Comment by ozkriff** [comment]
Merged, thanks!

**Pull request merged**